sample/ios/Podfile

```diff
@@ -13,7 +13,9 @@ target 'sample' do
   	:hermes_enabled => true
   )
 
-  pod 'RNSentry', :path => '../../RNSentry.podspec'
+  if !ENV['TEST']
+    pod 'RNSentry', :path => '../../RNSentry.podspec'
+  end
```

Comment:
why do we need that? does that mean that the `test` version don't include RNSentry?

Reply:
The test version has the pod auto-added by React Native, while we're developing using the SDK in the parent folder we need to add it manually.